ci/uninstall-thin-edge_io/uninstall-thin-edge_io.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+
+set -e
+
+usage() {
+    cat <<EOF
+USAGE:
+    delete-thin-edge_io [COMMAND]
+    
+COMMANDS:
+    remove     Uninstall thin-edge.io with keeping configuration files
+    purge      Uninstall thin-edge.io and also remove configuration files  
+
+EOF
+}
+
+stop_a_service_if_running() {
+    local status
+    status=$(systemctl show -p ActiveState --value "$1")
```

Comment:
Just FYI: `systemctl` has an `is-active` subcommand as well.

Reply:
Good idea to use this.